apps/src/dom.js

```diff
@@ -37,14 +37,14 @@ exports.getTouchEventName = function(eventName) {
 };
 
 var addEvent = function(element, eventName, handler) {
-  element.addEventListener(eventName, handler, false);
-
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
```

Comment:
Yeah, my next attempt is just changing it to hook touchstart/MSPointerDown/pointerdown in which case preventDefault should eat the event the first time.  Downside is that drag events out of that UI element will fire as a fast click, but on the plus side it means we now have fast click, and this should be localised to the elements that are bound this way.